gax/src/main/java/com/google/api/gax/retrying/NonCancellableFuture.java

```diff
@@ -46,15 +46,15 @@ public boolean cancel(boolean mayInterruptIfRunning) {
     return false;
   }
 
-  void cancelPrivately() {
+  public void cancelPrivately() {
     super.cancel(false);
   }
 
-  boolean setPrivately(ResponseT value) {
+  public boolean setPrivately(ResponseT value) {
```

Comment:
This does not look good, and it is not "privately" set as the method name suggests. Please check [ServerStreamingAttemptCallable.java#L128](https://github.com/googleapis/gax-java/pull/463/files#diff-cf2be21fc715df86e011ec7c6b968b0cR128)

Reply:
done